pyicloud/services/drive.py

```diff
@@ -113,8 +113,8 @@ def _update_contentws(self, folder_id, sf_info, document_id, file_object):
                 "is_executable": False,
                 "is_hidden": False,
             },
-            "mtime": int(time.time()),
-            "btime": int(time.time()),
+            "mtime": int(time.time()) * 1000,
+            "btime": int(time.time()) * 1000,
```

Comment:
I agree with @ixs

```suggestion
            "mtime": int(time.time() * 1000),
            "btime": int(time.time() * 1000),
```

Otherwise we will loose precision I think